src/controllers/userController.js

```diff
@@ -27,6 +28,13 @@ const getSingleUser = async (req, res) => {
 };
 
 const showCurrentUser = async (req, res) => {
+  // Function to get the current user's information
+  // Send a JSON response with the status code 200 OK and the current user's information
+  res.status(StatusCodes.OK).json({ user: req.user });
+};
```

Comment:
I think this also got caught up when resolving merge conflicts; the version of this function that we want is the one from main: https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/main/src/controllers/userController.js#L29-L37